cce/cce.go

```diff
@@ -83,12 +87,83 @@ func executeForLocal(hash string, buf []byte, h *Handler) error {
 		return nil
 	}
 	h.Status = status.Processing
-	// TODO unzip file and iterate through files
+	downloadedFiles, err := unzipDownloadedFiles(buf)
+	if err != nil {
+		return fmt.Errorf("falha ao descomprimir arquivos baixados, erro %q", err)
+	}
+	for _, file := range downloadedFiles {
+		csvReader := csv.NewReader(bufio.NewReader(file))
+		csvReader.Comma = ';'
+		csvReader.LazyQuotes = true
+		// TODO read lines and mount struct Descritor
+	}
 	return nil
 }
 
+func unzipDownloadedFiles(buf []byte) ([]*os.File, error) {
+	unzipDestination := "unzipped"
+	os.MkdirAll(unzipDestination, 0755)
+	extractAndWriteFile := func(f *zip.File) error {
```

Comment:
[Sugestão Pessoal]:  Coloca o conteúdo desse método dentro do for. Acho que dificultou mais a leitura do que ajudou.